BinTree/BinTree.h

```diff
@@ -19,18 +19,30 @@ class BinTree {
 public:
     BinTree();
     ~BinTree();
-    int getHeight();
-    bool insertNode(NodeData*);
+    BinTree(const BinTree& bt);
+    
+    int getHeight() const;
+    int getHeight (const NodeData &) const;
```

Comment:
getHeigh(NodeData&) is the method requested in the PDF. The other is not required.